Lets Do This/Views/Campaign/LDTCampaignDetailCampaignCell.m

```diff
@@ -41,19 +41,6 @@ - (void)awakeFromNib {
     self.staticInstructionLabel.text = @"When you’re done, submit a pic of yourself in action. #picsoritdidnthappen";
 }
 
-- (UICollectionViewLayoutAttributes *)preferredLayoutAttributesFittingAttributes:(UICollectionViewLayoutAttributes *)layoutAttributes {
-    UICollectionViewLayoutAttributes *attributes = [[super preferredLayoutAttributesFittingAttributes:layoutAttributes] copy];
-
-    [self setNeedsLayout];
-    [self layoutIfNeeded];
```

Comment:
In particular, honghaoz calls `layoutIfNeeded` [here](https://github.com/honghaoz/Dynamic-Collection-View-Cell-With-Auto-Layout-Demo/blob/7ea8b8152b6efef9aef740efd55702180e2f1cea/DynamicCollectionViewCellWithAutoLayout-Demo/ViewController.swift#L79) within `cellForItemAtIndexPath`.

The [`configCell` method](https://github.com/honghaoz/Dynamic-Collection-View-Cell-With-Auto-Layout-Demo/blob/7ea8b8152b6efef9aef740efd55702180e2f1cea/DynamicCollectionViewCellWithAutoLayout-Demo/MyCollectionViewCell.swift#L40) calls both `setNeedsLayout` and `layoutIfNeeded`

The original thread that inspired this PR however only calls `setNeedsLayout` and `layoutIfNeeded` in [`preferredLayoutSizeFittingSize`] (http://github.com/algal/UICVVariableHeightiOS8/blob/2a1924a5c204ecb2418700b75d44220519575a9b/UICVVariableHeightiOS8/MyCell.swift#L49) which is just called once the `sizeForItemAtIndexPath` method